volatility3/framework/symbols/windows/extensions/gui.py

```diff
@@ -2,13 +2,17 @@
 # which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
 #
 
-from typing import Optional, Tuple, Iterator
+import logging
+from typing import Optional, Tuple, Iterator, Generator
 
 from volatility3.framework import exceptions, constants, interfaces
 from volatility3.framework import objects
 from volatility3.framework.objects import utility
+from volatility3.framework.symbols.windows import extensions
 from volatility3.framework.symbols.windows.extensions import pool
 
+vollog = logging.getLogger(__name__)
+
 
 class tagWINDOWSTATION(objects.StructType, pool.ExecutiveObject):
```

Comment:
Can we put all the classes in this module inside a larger class (that can then be versioned)?  Otherwise changes to this require framework bumps.  I think it should be a catch-all, because no one's going to want to track the versions of each individual class in here.  We're trying to balance usability against upheaval from changes...

Reply:
Good call. Fixed now.